index.js

```diff
@@ -30,12 +30,20 @@ const main = async () => {
   server.log.info(`Server running at: ${address}`)
 
   for (const signal of ['SIGINT', 'SIGTERM']) {
-    process.on(signal, () =>
-      server.close().then(err => {
-        console.log(`close application on ${signal}`)
-        process.exit(err ? 1 : 0)
-      })
-    )
+    // Use once() so that double signals exits the app
+    process.once(signal, () => {
+      server.log.info({ signal }, 'closing application')
+      server
+        .close()
+        .then(() => {
+          server.log.info({ signal }, 'application closed')
+          process.exit(0)
```

Comment:
In general, we shouldn't be calling `process.exit()` like this. Instead, clean up the process normally and allow it to exit cleanly.